single-line-diagram-viewer/src/main/java/com/powsybl/sld/viewer/SingleLineDiagramViewer.java

```diff
@@ -1206,7 +1195,7 @@ private void updateLayoutsFactory(Network network) {
     }
 
     private void initStylesProvider() {
-        styles.put("Default", new DefaultDiagramStyleProvider());
+        styles.put("Default", new BasicStyleProvider());
         styles.put("Nominal voltage", null);
         styles.put("Topology", null);
```

Comment:
This is now the default style provider in SingleLineDiagram API, so could you add (default) to the text and put it first or make it selected by default?

Reply:
I meant TopologyStyleProvider is now the default style provider, not BasicStyleProvider